data/ecosystems/v/Vanity-ETH-Generator-Poly

```diff
@@ -0,0 +1,25 @@
+# Ecosystem Level Information
+title = "Vanity ETH Generator Poly"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/samaeleth"]
+
+# Repositories
+[[repo]]
+url = "https://github.com/samaeleth/vanity-eth-generator/blob/master/Poly-Vanity-ETH"
```

Comment:
These aren't repos.  You should just use url = "https://github.com/samaeleth/vanity-eth-generator"  one time.